src/lib.rs

```diff
@@ -143,16 +150,16 @@ pub fn select_coin_lowestlarger(
     let target = options.target_value + options.min_drain_value;
 
     let mut sorted_inputs: Vec<_> = inputs.iter().enumerate().collect();
-    sorted_inputs.sort_by_key(|(_, input)| effective_value(input, options.target_feerate));
+    sorted_inputs.sort_by_key(|(_, input)| effective_value(input, options.target_feerate).unwrap());
```

Comment:
Same thing here. No unwraps. Just include the error variant in the SeelctionError.

Reply:
Ah alright. Ya, that can be done too. But not needed now. Maybe just put a comment here as a TODO so we don't forget?

Resolving this now.